src/hdr_histogram.h

```diff
@@ -33,8 +33,27 @@
     using std::memory_order_seq_cst;
 
     using std::atomic_int_least64_t;
-#else /* for std C11 compiler */
-    #include <stdatomic.h>
+#else /* for std C compiler */
+    #ifdef _MSC_VER /* for MSVC as it dosn't support stdatomic.h */
+        #define WIN32_LEAN_AND_MEAN
+        #include <Windows.h>
+
+        typedef volatile int64_t atomic_int_least64_t;
+        #define memory_order_relaxed 0
+
+        #define atomic_load_explicit(loadAdd, memory_order) \
+            InterlockedCompareExchange64(loadAdd, 0, 0)
+        #define atomic_compare_exchange_weak_explicit( \
+                obj, expected, desired, succ, fail)    \
+            (InterlockedCompareExchange64(obj, desired, *expected) == *expected)
+        #define atomic_fetch_add_explicit(obj, arg, memory_order) \
+            InterlockedExchangeAdd64(obj, arg)
```

Comment:
Similar here (and others below) - I believe this is currently a full sequentially-consistent fence.

Reply:
Make comment refer to all memory orderings specified here